CMakeLists.txt

```diff
@@ -283,29 +283,52 @@ find_package (JPEG)
 ### --------------------------------------------------------------------
 ###  Some Guile related adjustements
 ### --------------------------------------------------------------------
-find_package (Guile REQUIRED)
-
-if (Guile_VERSION_STRING VERSION_LESS 1.6)
-  set (CONFIG_GUILE_SERIAL "A")
-  option (GUILE_A "Using Guile < 1.6.x" ON)
-  message (STATUS "Using GUILE_A: " ${Guile_VERSION_STRING})
-elseif (Guile_VERSION_STRING VERSION_LESS 1.8)
-  set (CONFIG_GUILE_SERIAL "B")
-  option (GUILE_B "Using Guile 1.6.x" ON)
-  message (STATUS "Using GUILE_B: " ${Guile_VERSION_STRING})
-elseif (Guile_VERSION_STRING VERSION_LESS 2.0)
+
+# Find Guile and determine which version we are using.
```

Comment:
this Guile related code could be maintained in `cmake/FindGuile.cmake`

Reply:
Thanks, I agree.

Reply:
It seems that searching for different versions of guile can be done with one call of `pkg_search_module` (see commit 99477d8). I guess then there is no need to put it in a separate file, or would you put the following lines in there as well?

Reply:
It seems this can all be done in one line with `pkg_search_module` (see 99477d8). I guess then there's not much need for a separate file.

Reply:
```
diff --git a/CMakeLists.txt b/CMakeLists.txt
index 2ae03a041..52056008d 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -549,7 +549,8 @@ set (TeXmacs_MacOS_SRCS
 ### --------------------------------------------------------------------

 set (TeXmacs_Libraries
-  ${Guile_LIBRARIES}
+  /usr/local/lib/libguile.dylib
+  # ${Guile_LIBRARIES}
   ${ZLIB_LIBRARIES}
   ${JPEG_LIBRARIES}
   ${PNG_LIBRARIES}
```

I need to make the above diff to make it link successfully. Otherwise, I will get the following error:
```
ld: library not found for -lguile
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

Reply:
It's strange, I would expect `/usr/local/lib` to be detected by pkg-config.

This has been in the master for a while now. Has anything changed in your setup?